src/polycap.h

```diff
@@ -0,0 +1,105 @@
+// Libraries to include
```

Comment:
Publieke headers van een library mogen eigenlijk alleen maar zaken bevatten die essentieel zijn om voor een ander programma je geëxporteerde functies te kunnen gebruiken. Al de rest moet blijven in de implementatie files (`polycap.c`). Als het trouwens niet nodig is dat anderen aan de inhoud van je structs kunnen om te kunnen werken, verhuis je trouwens best ook de definitie ervan naar de implementatie, en hou je enkel een declaratie ervan over in de header zoals:

````c
struct inp_file;
````

Als je geëxporteerde functies enkel pointers naar deze structs bevatten is dit mogelijk, anders niet...